### PR TITLE
Use new liServer.register methods

### PR DESCRIPTION
**Pull request by peyerluk**
# Changelog

- 🎁 Use new `liServer.register...()` methods everywhere.
- 🎁 Replace `liServer.features.register()` with `liServer.registerServerRoutes()`
- 🔧 Use `daily-planet` as example project handle

### CODE REVIEW

File: content/customising/server/server-hooks.md [hunk 1]
```diff
@@ -69,13 +69,14 @@ const appConfig = require('./conf')
...
-  liServer.features.api('li-documents').registerPublicationServerHooks({
+  // Global publish hooks
+  liServer.registerGlobalPublicationHooks({
     async preparePublishHookAsync ({documentVersion}) { return }
   })
 
-  liServer.features.api('li-documents').registerPublicationHooks({
-    projectHandle: 'your-awesome-project',
-    channelHandle: 'default',
+  // Proejct-specific publish hooks
```
Comment (DaRaFF) [inline content/customising/server/server-hooks.md:77]:
```suggestion
  // Project-specific publish hooks
```

File: content/guides/integrations/desknet/index.md [hunk 2]
```diff
@@ -632,7 +632,7 @@ To begin with you should register a new create function in your server runtime c
 ```js
 liServer.registerInitializedHook(async () => {
   const documentApi = liServer.features.api('li-documents').document
-  documentApi.registerCreateFunction({
+  documentApi.registerGenerateFunction({
```
Comment (DaRaFF) [inline content/guides/integrations/desknet/index.md:635]: Is that API correct?

**Review by DaRaFF — APPROVED** [review]
Really nice, I like it!

2 issues I see at the moment:
1) In the documentation there should be a link between the old and the new API (as I wrote in that comment - https://github.com/livingdocsIO/documentation/pull/758/files#r1382338621)
2) The new API is available with release-2023-11. This means all other customers with old version could be puzzled

File: content/guides/integrations/desknet/index.md (hunk 2 above)
Comment (peyerluk) [reply content/guides/integrations/desknet/index.md:635]: Yes :)

**Pull request merged**